### PR TITLE
Name change

### PR DESCRIPTION
**Pull request by Tess-LaCoil**
Updated name to hmDE - hierarchical methods for DEs - from rmot as 'repeat measures' is already a thing in statistics.

### CODE REVIEW

**Comment by dfalster** [comment]
Looks promising. One thought - will it be annoying to have a name mixing caps and lowercase? Thinking as a user, it would be easier if functions names didn't include case change. WDYT?

**Comment by Tess-LaCoil** [comment]
I'm ambivilent on the matter. If you'd prefer all lower/upper say which and I'll make the changes.

**Comment by dfalster** [comment]
Thoughts @fontikar ?

**Comment by fontikar** [comment]
I'm a big fan of snake_case rather than camelCase, easier to type as @dfalster mentioned. Not the biggest fan of a combo of snake and camel 

e.g. Found in NAMESPACE of function names.

```
export(hmDE_assign_data)
export(hmDE_model)
export(hmDE_run)
```

is `hmde` a better alternative?

**Comment by fontikar** [comment]
Tests failing because of the new data objects: 

```
❯ checking for missing documentation entries ... WARNING
  Undocumented code objects:
    ‘hmDE_assign_data’ ‘hmDE_model’ ‘hmDE_run’ ‘Lizard_Mass_Data’
    ‘Small_Tree_Size_Data’ ‘Tree_Size_Data’ ‘Trout_Size_Data’
  Undocumented data sets:
    ‘Lizard_Mass_Data’ ‘Small_Tree_Size_Data’ ‘Tree_Size_Data’
    ‘Trout_Size_Data’
  All user-level objects in a package should have documentation entries.
  See chapter ‘Writing R documentation files’ in the ‘Writing R
  Extensions’ manual.
  ```
  
  Are these internal testing data or data the user can access to take the package for a spin.
  
  See here on how to document: https://r-pkgs.org/data.html

**Comment by Tess-LaCoil** [comment]
Okay, sounds like hmde is the go. I'll implement it when I'm home from Wollongong.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Package: rmot
+Package: hmde
 Title: What the Package Does (One Line, Title Case)
```
Comment (dfalster) [inline DESCRIPTION:2]: Missing a title and need to update version number

**Review by dfalster — CHANGES_REQUESTED** [review]
Name looks good, but lost package title and version number. Please update

**Review by dfalster — APPROVED** [review]

**Pull request merged**